OpenSim/Simulation/SimbodyEngine/Joint.cpp

```diff
@@ -737,15 +737,6 @@ int Joint::assignSystemIndicesToBodyAndCoordinates(
 
         // ONLY the base Joint can do this assignment
         mobilized->setMobilizedBodyIndex(mobod.getMobilizedBodyIndex());
-
```

Comment:
This code is no longer necessary, because calling `setMobilizedBodyIndex` now implies that transitive indices are handled by each frame implementation correctly.